uaclient/config.py

```diff
@@ -77,12 +77,7 @@
 
 
 class UAConfig:
-    data_paths = {
-        "instance-id": DataPath("instance-id", True),
-        "machine-access-cis": DataPath("machine-access-cis.json", True),
-        "lock": DataPath("lock", False),
-        "status-cache": DataPath("status.json", False),
-    }  # type: Dict[str, DataPath]
+    data_paths = {}  # type: Dict[str, DataPath]
```

Comment:
Do we even need to keep this variable around?

Reply:
Yes, we do. There are some "hidden" state files on the system still and for the config module to handle them, we need the `data_paths`. I am getting rid of it on the following PR #2966